gradle/versions.gradle

```diff
@@ -20,7 +20,7 @@ ext {
     junitVersion = "4.13.1"
     restAssuredVersion = "4.3.0"
     javaxValidationApiVersion= "2.0.1.Final"
-    explorerApiCommonVersion = "1.1.13"
+    explorerApiCommonVersion = "1.1.14-UPGRADE-GSON-V1-SNAPSHOT"
```

Comment:
The SNAPSHOT version is intentional?

Reply:
This will be changed once I release the common lib, included here to test the common lib changes in integration tests.